packages/reason-expo/src/Svg.re

```diff
@@ -167,6 +167,7 @@ module G = {
       ~fill: string=?,
       ~stroke: string=?,
       ~strokeWidth: string=?,
+      ~transform: string=?,
```

Comment:
are you adding or removing transform?

Reply:
Looks like it should be there and it in both versions. Maybe it had been removed erroneously and re-added.